grpc/client.go

```diff
@@ -484,6 +516,33 @@ func (c *Client) parseConnectParams(raw map[string]interface{}) (connectParams,
 	return params, nil
 }
 
+func parseConnectConnectionSharingParam(params *connectParams, v interface{}) error {
+	var (
+		ok                    bool
+		connectionSharingBool bool
```

Comment:
Out of curiosity, why do you think mixing the param type is beneficial? Why not stick with the number?